src/services/recim/team.ts

```diff
@@ -50,12 +50,14 @@ export type TeamMatchHistory = {
 };
 
 export type TeamRecord = {
-  ID: number;
+  SeriesID: number;
```

Comment:
this was originally just straight up wrong. I don't know how I didn't catch it. I suppose it's because we don't use the seriesID for anything right now